packages/pwa-kit-react-sdk/src/ssr/universal/contexts/index.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React, {useEffect, useRef} from 'react'
+import PropTypes from 'prop-types'
+import {useLocation} from 'react-router-dom'
+import DeviceContext from '../device-context'
+
+const ExpressContext = React.createContext()
```

Comment:
Did you mean to copy this over?

Reply:
No, it is not meant to be there. I've removed it